sympy/combinatorics/pc_groups.py

```diff
@@ -0,0 +1,441 @@
+            G = self.pc_series[i]
+            H = self.pc_series[i+1]
+            rel_orders.append(G.order() // H.order())
+        return rel_orders
+
+    def is_prime_order(self):
+        for order in self.relative_order():
+            if order not in sieve:
+                return False
+        return True
+
+    def length(self):
+        return len(self.pcgs)
+
+    def exponent_vector(self, element, group):
+        G = PermutationGroup()
+        for g in self.pcgs:
+            G = PermutationGroup(G.generators + [g])
+        gens = G.generator_product(element, original = True)
+
+        perm_to_free = {}
+        for sym, g in zip(group.generators, self.pcgs):
+            perm_to_free[g] = sym
+        w = group.identity
+        for g in gens:
+            w = w*perm_to_free[g]
+
+        pc_presentation = self.pc_presentation(group)
+        collector = Collector(pc_presentation, self.relative_order(), group)
+        word = collector.collected_word(w)
+
+        index = {s: i for i, s in enumerate(group.symbols)}
+        exp_vector = [0]*len(group)
+        word = word.array_form
+        for t in word:
+            exp_vector[index[t[0]]] = t[1]
+        return exp_vector
+
+    def power_relations(self, group):
+        """
+        Return a list of power relations.
+        If `x` is a free group element and `re` represents
+        the relative order of `x` then power relation of a
+        single generator `x` is `x**re`.
+
+        Examples
+        ========
+        >>> from sympy.combinatorics.pc_groups import PolycyclicGroup
+        >>> from sympy.combinatorics.named_groups import SymmetricGroup
+        >>> from sympy.combinatorics.free_groups import free_group
+        >>> F, x0, x1 = free_group("x0, x1")
+        >>> G = SymmetricGroup(4)
+        >>> pc_group = G.polycyclic_group()
+        >>> group = F
+        >>> pc_group.power_relations(group)
+        [x1**3, x0**2]
+
+        """
+        power_relators = []
+        for i, s in enumerate(group.symbols):
+            gen = ((s, self.relative_order()[i]), )
+            gen = group.dtype(gen)
+            power_relators.append(gen)
+        power_relators.reverse()
+        return power_relators
+
+    def conjugate_relations(self, group):
+        """
+        Return a list of conjugate relations.
+        Conjugate relations of a polycyclic group are of the
+        form `x[i]**-1*x[i+1]*x[i]`.
+
+        Examples
+        ========
+        >>> from sympy.combinatorics.pc_groups import PolycyclicGroup
+        >>> from sympy.combinatorics.named_groups import SymmetricGroup
+        >>> from sympy.combinatorics.free_groups import free_group
+        >>> F, x0, x1, x2, x3 = free_group("x0, x1, x2, x3")
+        >>> G = SymmetricGroup(4)
+        >>> pc_group = G.polycyclic_group()
+        >>> group = F
+        >>> pc_group.conjugate_relations(group)
+        [x2**-1*x3*x2, x1**-1*x3*x1, x1**-1*x2*x1, x0**-1*x3*x0,
```

Comment:
Also here we only have the lhs expressions of the relations, not the actual relations.